tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -93,7 +93,7 @@ public static string GetMonitoringHomePath()
         {
             var monitoringHomeDirectoryEnvVar = "MonitoringHomeDirectory";
             var monitoringHome = Environment.GetEnvironmentVariable(monitoringHomeDirectoryEnvVar);
-            if (!string.IsNullOrEmpty(monitoringHome))
+            if (string.IsNullOrEmpty(monitoringHome))
```

Comment:
This is already in master, you might need to rebase

Reply:
Good point, done.